### DISCUSSION TITLE
Can't resolve "ValueError: Unable to configure formatter 'airflow_coloured'" issue

### DISCUSSION THREAD

**Discussion by slycyberguy** [apache/airflow#38056] [General]
I'm trying to run a couple unit tests such as `tests/providers/amazon/aws/operators/test_s3.py` using pytest, but I get the output (below) which I can't resolve. I looked into `airflow_local_settings.py`, but I'm not sure why 'airflow_coloured' formatter can't be configured. Any help will be great. Thanks.

```
ImportError while loading conftest '...airflow/dev/airflow/tests/providers/conftest.py'.
../../../conftest.py:24: in <module>
    from airflow.models import Connection
../../../../../airflow/__init__.py:56: in <module>
    settings.initialize()
../../../../../airflow/settings.py:531: in initialize
    LOGGING_CLASS_PATH = configure_logging()
../../../../../airflow/logging_config.py:74: in configure_logging
    raise e
../../../../../airflow/logging_config.py:69: in configure_logging
    dictConfig(logging_config)
/usr/lib/python3.11/logging/config.py:823: in dictConfig
    dictConfigClass(config).configure()
/usr/lib/python3.11/logging/config.py:555: in configure
    raise ValueError('Unable to configure '
E   ValueError: Unable to configure formatter 'airflow_coloured'
```

**Comment by slycyberguy** [comment]
Can anyone help or need more clarification?

**Reply by Taragolis** [reply]
Looks works fine in Breeze Shell

```console
❯ breeze shell --backend postgres

root@21bb00060ad3:/opt/airflow# pytest tests/providers/amazon/aws/operators/test_s3.py

...

================================================================= 42 passed, 2 warnings in 5.88s =================================================================
```

**Comment by potiuk** [comment]
I would not be surprised if you try to run the tests on windows to get such an error

**Reply by vincbeck** [reply]
I have the same error with my mac os M1

**Reply by potiuk** [reply]
Likely you need to upgrade your venv /breeze to install latest version of packages `pip install -e ".[devel]" --upgrade --upgrade-stategy eager` or equivalent. Generally speaking `breeze ci-image build` does it automatically but when you have older versions of dependencies installed in your local venv, various problems like this might happen.

**Reply by potiuk** [reply]
This is what `breeze` does automatically when you rebuild the image, and this is where your local venv should be maintained manually and generally when you see strange problems like this, first thing to do is to check if you have all the dependencies installed and reinstall them.

**Reply by potiuk** [reply]
(but I am guessing now of course - the fact that you see that afer holidays @vincbeck indicates that this could be the issue).

**Reply by vincbeck** [reply]
Good guess! It resolved the issue!

**Comment by Lukas1v** [comment]
If you're getting this error while unit testing in the devcontainer, manually re-pulling the docker image and rebuilding the container solved it for me.
